app/models/foreman_openscap/policy.rb

```diff
@@ -248,14 +249,18 @@ def ensure_period_specification_present
 
     def assign_policy_to_hostgroups
       if hostgroups.any?
-        puppetclass = Puppetclass.find_by_name(SCAP_PUPPET_CLASS)
+        puppetclass = find_scap_puppetclass
         hostgroups.each do |hostgroup|
           hostgroup.puppetclasses << puppetclass unless hostgroup.puppetclasses.include? puppetclass
           populate_overrides(puppetclass, hostgroup)
         end
       end
     end
 
+    def find_scap_puppetclass
```

Comment:
It seems that this method is used only once. so not sure why to create it (unless plans to use more of it?)